operator/config/gke-addon/manager_patch.yaml

```diff
@@ -29,4 +29,7 @@
   value:
     postStart:
       exec:
-        command: ["./gke_addon_poststart"]
\ No newline at end of file
+        command: ["./gke_addon_poststart"]
+- op: add
+  path: /spec/template/spec/containers/0/securityContext/readOnlyRootFilesystem
```

Comment:
We can also turn on the readOnlyRootFilesystem for all KCC release approaches (manual, CC, gke add-on). But I'd prefer do it step by step